client/less/main.less

```diff
@@ -1175,6 +1175,11 @@ and (max-width : 400px)  {
   font-size: 17px;
 }
 
+// Challange editor CSS for iOS.  Issue 9104.
```

Comment:
@craigstroman you just need to remove the comment line

`// Challange editor CSS for iOS.  Issue 9104.`

this is not necessary.